code/game/objects/items/robot/cyborg_gripper.dm

```diff
@@ -0,0 +1,220 @@
+    protections to try and eliminate the chance of a hypothetical synthetic uprising, not wishing to see a repeat of the IPC uprising in 2525."
+
+
+/obj/item/gripper/Initialize(mapload)
+	. = ..()
+	can_hold = typecacheof(can_hold)
+
+/obj/item/gripper/ui_action_click(mob/user)
+	drop_gripped_item()
+
+/obj/item/gripper/proc/drop_gripped_item(silent = FALSE)
+	if(!gripped_item)
+		return
+	if(!silent)
+		to_chat(loc, "<span class='warning'>You drop [gripped_item].</span>")
+	gripped_item.forceMove(get_turf(src))
+	gripped_item = null
+
+/obj/item/gripper/attack_self(mob/user)
+	if(!gripped_item)
+		to_chat(user, "<span class='warning'>[src] is empty.</span>")
+		return
+	gripped_item.attack_self(user)
+
+/obj/item/gripper/attack(mob/living/carbon/M, mob/living/carbon/user)
+	return
+
+/// Grippers are snowflakey so this is needed to to prevent forceMoving grippers after `if(!user.drop_item())` checks done in certain attackby's. // What does this even MEAN - GDN
+/obj/item/gripper/forceMove(atom/destination)
+	return
+
+/obj/item/gripper/afterattack(atom/target, mob/living/user, proximity, params)
+	if(!target || !proximity) //Target is invalid or we are not adjacent.
+		return FALSE
+
+	if(ishuman(target) && can_help_up)	// Shake people awake, get them on their feet.
+		var/mob/living/carbon/human/pickup_target = target
+		if(!IS_HORIZONTAL(pickup_target))
+			return
+		pickup_target.AdjustSleeping(-10 SECONDS)
+		pickup_target.AdjustParalysis(-6 SECONDS)
+		pickup_target.AdjustStunned(-6 SECONDS)
+		pickup_target.AdjustWeakened(-6 SECONDS)
+		pickup_target.AdjustKnockDown(-6 SECONDS)
+		pickup_target.stand_up()
+		playsound(user.loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+		user.visible_message( \
+			"<span class='notice'>[user] shakes [pickup_target] trying to wake [pickup_target.p_them()] up!</span>",\
+			"<span class='notice'>You shake [pickup_target] trying to wake [pickup_target.p_them()] up!</span>",\
+			)
+		return
+
+	if(gripped_item) //Already have an item.
+
+		//Pass the attack on to the target. This might delete/relocate gripped_item.
+		if(!target.attackby(gripped_item, user, params))
+			// If the attackby didn't resolve or delete the target or gripped_item, afterattack
+			// (Certain things, such as mountable frames, rely on afterattack)
+			gripped_item?.afterattack(target, user, 1, params)
+
+		//If gripped_item either didn't get deleted, or it failed to be transfered to its target
+		if(!gripped_item && contents.len)
+			gripped_item = contents[1]
+			return FALSE
+		else if(gripped_item && !contents.len)
+			gripped_item = null
+
+	else if(isitem(target)) //Check that we're not pocketing a mob.
+		var/obj/item/I = target
+		if(is_type_in_typecache(I, can_hold)) // Make sure the item is something the gripper can hold
+			to_chat(user, "<span class='notice'>You collect [I].</span>")
+			I.forceMove(src)
+			gripped_item = I
```

Comment:
Should this return `TRUE`?

Reply:
I don't know and I am scared to touch it.

Reply:
It now returns true. Nothing broke.